python-shell/src/generate.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+from gafferpy.gaffer_connector import GafferConnector
+from fishbowl.fishbowl import Fishbowl
+
+# Generate the core api using spring-rest as it has access to every store operation
+gc = GafferConnector("http://localhost:8080/rest")
```

Comment:
Are you missing a /version?

Reply:
See above

Reply:
Might it be worth updating the doc provided with the GafferConnector class to address this?

Reply:
I suppose there is this: https://github.com/gchq/gaffer-tools/blob/7d90f3f3928e7c5230cf55dee9677107931c1a75/python-shell/src/gafferpy/gaffer_connector.py#L34-L40
But no explanation that the spring rest doesn't require version.